.github/workflows/documentation.yml

```
@@ -0,0 +1,41 @@
+name: Documentation
+on:
+  push:
+    branches:
+    - main
```

If you run on pushes to main the documentation won't match the released version.

```suggestion
    tags:
      - v*
```

---

Changed the trigger for tags